### PR TITLE
dynamically add hq code root to python path

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: apps/deploy/app_config.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+def _add_source_root(source_root):
+    sys.path.append(source_root)
+
+
+class HqDeploy(AppConfig):
+    name = "apps.deploy"
+
+    def ready(self):
+        # sneaky trick to add the HQ source onto the python path
+        _add_source_root(os.path.join(settings.HQ_SOURCE_ROOT))
```
Comment (snopoke) [inline apps/deploy/app_config.py:16]: @benrudolph not sure what you think about this approach

Comment (benrudolph) [inline apps/deploy/app_config.py:16]: interesting so this is so we can use modules in HQ without actually loading HQ as a submodule. sounds reasonable to me. the reason we'd do this over a submodule is for simplicity and so we could possibly just point this to `/home/cchq/www/env/current` and thus make it faster/simpler to deploy chief?

Comment (benrudolph) [inline apps/deploy/app_config.py:16]: also loading hq as a submodule doesn't really feel right

File: apps/deploy/templates/chief/status.html [hunk 2]
```diff
@@ -12,9 +12,12 @@
 
 {% block js-inline %}{{ block.super }}
 <script type="text/javascript">
-    window.Chief.ReleasesUrl ='{% url 'chief_releases' %}';
-    window.Chief.SubmodulesUrl = '{% url 'chief_submodules' %}';
-    window.Chief.PrepareUrl = '{% url 'chief_prepare' %}';
+    window.Chief.config = {
```
Comment (benrudolph) [inline apps/deploy/templates/chief/status.html:15]: :+1:

**Pull request merged**